lib/accelerators/kd_tree.hpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Cameron Custer on 11/8/22.
+//
+
+#ifndef CPPTRACE_LIB_HITTABLE_KD_TREE_HPP_
+#define CPPTRACE_LIB_HITTABLE_KD_TREE_HPP_
+
+#include "../common/common.hpp"
+#include "../common/ray.hpp"
+#include "../hittable/bounding/aabb.hpp"
+#include "../hittable/hit.hpp"
+#include "../hittable/hittable.hpp"
+#include "../hittable/hittable_list.hpp"
+
+struct KDTree : Hittable {
```

Comment:
`struct KDTree : public Hittable {`

Reply:
Why should it be different than what BVH has?